stl/inc/random

```diff
@@ -4001,12 +4028,18 @@ private:
         _Ty _Px;
         _Ty _Vx1;
         _Ty _Vx2;
+        const _Ty _Vx3{1};

```

Comment:
> I kind of feel like storing the constant `1` in an obscurely named variable is less illuminating than just writing out. `_Ty{1}` whenever you need it. No change requested though. I don't feel super strongly.

I use it in two places. 

https://github.com/microsoft/STL/blob/109ec2ac050e2ba5108f1dee6a483edfaff35074/stl/inc/random#L4037-L4042

That's because if the value in the formula changed for any reason, then we should take care of that value in the `if` statement as well. So, I thought it is better to have it as a `const variable`